### PR TITLE
test and some other small improvements

### PR DESCRIPTION
**Pull request by PanderMusubi**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,9 +1,26 @@
-To test: modify docker-compose-local.yml to your own local music dir
+# Summary
 
-# build and run container
+This repository is a resty Docker container which offers a website with music
```
Comment (dromer) [inline README.md:3]: `Openresty` then.

Also you don't have to use it for music. It's more like a customized "fancy-index" -> https://www.nginx.com/resources/wiki/modules/fancy_index/

File: templates/head.html [hunk 2]
```diff
@@ -12,6 +12,6 @@
                 <img src="/img/hotmixeswhite.png" class="hotmixlogo"/>
             </a>
         </header>
-        <h1 class="h1 h1mobile">Tons Of Mix - Downloads & Streams (right click + save as for downloads)<br>
-            Welcome to Hotmixes.net (Beta) - The CBS / Intergalactic FM Archives<br>
+        <h1 class="h1 h1mobile">Tons Of Mix - Downloads & Streams (right click + Save link as... for downloads)<br>
```
Comment (dromer) [inline templates/head.html:15]: This content is `ad verbum` how the boss wanted it.

Ask him if he wants to change this.

File: README.md [hunk 3]
```diff
@@ -1,9 +1,26 @@
...
-# access container
+Install `docker-compose`, e.g. with
 
-http://localhost:8080
+    sudo apt-get install docker-compose
+
+# Configuration
+
+For testing, modify `docker-compose-local.yml` to find MP3 files on your local
+machine. Alternatively, create a directoru `~/Music/alfa` with subdirectores
```
Comment (dromer) [inline README.md:15]: `directory`

Also this folder needs to have folders named `0-9` and `a` through `z`.

**Review by dromer — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (dromer) [reply README.md:15]: Something like:
```
mkdir 0-9; for i in {a..z}; do mkdir $i; done
```

File: templates/head.html (hunk 2 above)
Comment (PanderMusubi) [reply templates/head.html:15]: Got an okay on this one.

File: README.md (hunk 1 above)
Comment (PanderMusubi) [reply README.md:3]: Thanks. Fixed in next commit.

File: README.md (hunk 3 above)
Comment (PanderMusubi) [reply README.md:15]: Thanks, also fixed in next commit.

**Pull request merged**